src/main/webapp/WEB-INF/index.html

```diff
@@ -0,0 +1,22 @@
+<!DOCTYPE html>
```

Comment:
Do we need this file? As you also created src/main/webapp/index.html

Reply:
NOO WE DONT NEED THIS FILE